internal/command/deploy/deploy.go

```diff
@@ -60,10 +60,7 @@ var CommonFlags = flag.Set{
 		Shorthand:   "e",
 		Description: "Set of environment variables in the form of NAME=VALUE pairs. Can be specified multiple times.",
 	},
-	flag.Bool{
-		Name:        "auto-confirm",
-		Description: "Will automatically confirm changes when running non-interactively.",
-	},
+	flag.Yes().SetDescription("Will automatically confirm changes when running non-interactively."),
```

Comment:
Is there a benefit to changing the flag description for specific commands, or can we use the same flag description for everything? eg "Skip confirmation prompt"

Reply:
No not necessarily, I mostly just changed it so this didn't change much, but honestly it doesn't matter much.